TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,7 +104,7 @@ public function initializeArguments()
     /**
      * Renders an HTML img tag with a thumbnail image, created from a given image.
      *
-     * @param ImageInterface $image The image to be rendered as an image
+     * @param ImageInterface|null $image The image to be rendered as an image
```

Comment:
Update: Now it failed when the image was empty ofc, changed the type to mixed (because I noticed that the image can also be an empty string after being removed from within the backend).

regression:
https://github.com/neos/neos-development-collection/pull/1203/commits/8cde724a22ed65b8d4d5ac4cc2ee9290f429170c